diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index f26dab8..e37edd5 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -24,8 +24,8 @@ set(OPTIONS
   "-DVIAME_ENABLE_SMQTK=ON"
   "-DVIAME_ENABLE_PYTORCH=OFF"
   "-DVIAME_ENABLE_DIVE=OFF"
-  "-DVIAME_ENABLE_GDAL=OFF"
-  "-DVIAME_ENABLE_VIVIA=OFF"
+  "-DVIAME_ENABLE_GDAL=ON"
+  "-DVIAME_ENABLE_VIVIA=ON"
   "-DVIAME_ENABLE_DARKNET=OFF"
   "-DVIAME_ENABLE_SCALLOP_TK=OFF"
   "-DVIAME_DOWNLOAD_MODELS=OFF"
