diff --git a/cmake/build_server_windows.cmake b/cmake/build_server_windows.cmake
index 3016b2eee..06056dd58 100644
--- a/cmake/build_server_windows.cmake
+++ b/cmake/build_server_windows.cmake
@@ -36,7 +36,7 @@ set(OPTIONS
   "-DVIAME_ENABLE_PYTORCH-NETHARN=ON"
   "-DVIAME_ENABLE_PYTORCH-PYSOT=ON"
   "-DVIAME_ENABLE_PYTORCH-SAM=ON"
-  "-DVIAME_ENABLE_PYTORCH-ULTRALYTICS:BOOL=OFF"
+  "-DVIAME_ENABLE_PYTORCH-ULTRALYTICS:BOOL=ON"
   "-DVIAME_ENABLE_SCALLOP_TK=OFF"
   "-DVIAME_ENABLE_SMQTK=ON"
   "-DVIAME_ENABLE_VIVIA=ON"
