diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index 54b0dd9..5e093f7 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -28,7 +28,7 @@ set(OPTIONS
   "-DVIAME_ENABLE_PYTORCH-MMDET=ON"
   "-DVIAME_ENABLE_PYTORCH-NETHARN=ON"
   "-DVIAME_ENABLE_PYTORCH-PYSOT=ON"
-  "-DVIAME_ENABLE_DIVE=OFF"
+  "-DVIAME_ENABLE_DIVE=ON"
   "-DVIAME_ENABLE_GDAL=OFF"
   "-DVIAME_ENABLE_VIVIA=OFF"
   "-DVIAME_ENABLE_DARKNET=ON"
