diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index f26dab8..1f76460 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -22,7 +22,12 @@ set(OPTIONS
   "-DVIAME_ENABLE_PYTHON=ON"
   "-DVIAME_ENABLE_PYTHON-INTERNAL=ON"
   "-DVIAME_ENABLE_SMQTK=ON"
-  "-DVIAME_ENABLE_PYTORCH=OFF"
+  "-DVIAME_ENABLE_PYTORCH=ON"
+  "-DVIAME_ENABLE_PYTORCH-INTERNAL=OFF"
+  "-DVIAME_ENABLE_PYTORCH-VIS-INTERNAL=OFF"
+  "-DVIAME_ENABLE_PYTORCH-MMDET=ON"
+  "-DVIAME_ENABLE_PYTORCH-NETHARN=ON"
+  "-DVIAME_ENABLE_PYTORCH-PYSOT=ON"
   "-DVIAME_ENABLE_DIVE=OFF"
   "-DVIAME_ENABLE_GDAL=OFF"
   "-DVIAME_ENABLE_VIVIA=OFF"
