diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index 016127212..b9e231e81 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -16,7 +16,7 @@ set(OPTIONS
   "-DVIAME_FIXUP_BUNDLE=ON"
   "-DVIAME_ENABLE_CUDNN=ON"
   "-DVIAME_ENABLE_CUDA=ON"
-  "-DVIAME_ENABLE_DARKNET=OFF"
+  "-DVIAME_ENABLE_DARKNET=ON"
   "-DVIAME_ENABLE_DIVE=OFF"
   "-DVIAME_ENABLE_FFMPEG=ON"
   "-DVIAME_ENABLE_GDAL=OFF"
