diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index 718d050..e71a201 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -28,9 +28,11 @@ set(OPTIONS
   "-DVIAME_ENABLE_VIVIA=OFF"
   "-DVIAME_ENABLE_DARKNET=OFF"
   "-DVIAME_ENABLE_SCALLOP_TK=OFF"
+  "-DVIAME_ENABLE_SEAL=ON"
   "-DVIAME_DOWNLOAD_MODELS=ON"
   "-DVIAME_BUILD_KWIVER_DIR=C:/tmp/kv5"
   "-DVIAME_BUILD_PLUGINS_DIR=C:/tmp/vm5"
+  "-DVIAME_FLETCH_BUILD_DIR=C:/tmp/fl5"
 )
 
 set(platform Windows10)
