diff --git a/cmake/build_server_windows_msi.cmake b/cmake/build_server_windows_msi.cmake
index 5e093f7..a6a3ef1 100644
--- a/cmake/build_server_windows_msi.cmake
+++ b/cmake/build_server_windows_msi.cmake
@@ -29,8 +29,8 @@ set(OPTIONS
   "-DVIAME_ENABLE_PYTORCH-NETHARN=ON"
   "-DVIAME_ENABLE_PYTORCH-PYSOT=ON"
   "-DVIAME_ENABLE_DIVE=ON"
-  "-DVIAME_ENABLE_GDAL=OFF"
-  "-DVIAME_ENABLE_VIVIA=OFF"
+  "-DVIAME_ENABLE_GDAL=ON"
+  "-DVIAME_ENABLE_VIVIA=ON"
   "-DVIAME_ENABLE_DARKNET=ON"
   "-DVIAME_ENABLE_SCALLOP_TK=OFF"
   "-DVIAME_DOWNLOAD_MODELS=ON"
